src/lm_buddy/jobs/_entrypoints/lm_harness.py

```diff
@@ -1,43 +1,45 @@
 from typing import Any
 
 import lm_eval
+import pandas as pd
 import torch
 from lm_eval.models.huggingface import HFLM
 from lm_eval.models.openai_completions import OpenaiCompletionsLM
 
 from lm_buddy.integrations.huggingface import (
     AutoModelConfig,
     HuggingFaceAssetLoader,
-    HuggingFaceAssetPath,
     resolve_peft_and_pretrained,
 )
 from lm_buddy.integrations.wandb import (
     ArtifactLoader,
     ArtifactType,
+    WandbArtifactConfig,
     WandbResumeMode,
     build_table_artifact,
     default_artifact_name,
     wandb_init_from_config,
 )
-from lm_buddy.jobs.common import LMBuddyJobType
+from lm_buddy.jobs.common import EvaluationResult, LMBuddyJobType
 from lm_buddy.jobs.configs import LMHarnessJobConfig, LocalChatCompletionsConfig
+from lm_buddy.paths import LoadableAssetPath
 
 
-def get_numeric_metrics(
+def get_per_task_dataframes(
     results: dict[str, dict[str, Any]],
-) -> dict[str, list[tuple[str, float]]]:
-    """Filter non-numeric values from the evaluation results.
+) -> dict[str, pd.DataFrame]:
```

Comment:
Returning these as `pd.DataFrame` now since the type is a bit cleaner to work with when included on the `EvaluationResult` class.